aiter/ops/triton/mha.py

```diff
@@ -1,9 +1,17 @@
+import os
 import torch
 import triton
 import triton.language as tl
 
 from typing import Optional, Tuple
 
+# env vars
+USE_INT64_STRIDES = os.environ.get("USE_INT64_STRIDES", "1").lower() in (
+    "1",
+    "true",
+    "yes",
+)
```

Comment:
@micmelesse  We don't want to have env vars in AITER. Can you please set USE_INT64_STRIDES=0(default value) and then add a function to override/set it.  This way the user of the op(like VLLM framework, etc) can set it. If they want to have it controlled through env var, then they can add it on their end.

By the way, we use a similar approach for MoE op.

Reply:
I can remove the env variable